.github/workflows/ci.yml

```diff
@@ -316,9 +316,7 @@ jobs:
         path: ${{ env.ARTIFACT }}
 
 
-  # This section is for tests on ubuntu 16 (and on ubuntu 20).
-  # The tests with Xvfb in this section work fine on ubuntu 16 and on Ubuntu 20, but they don't work on Ubuntu 18.
-  # (Both 18.04 & 20.04 have the same Open GL extension support, but the SDL library fails to initialize for 18.04)
+  # This section is for tests on ubuntu 20 (and on ubuntu 22).
```

Comment:
Maybe remove the version numbers from the comment completely? (Or remove the complete comment completely?)